src/systems/ec/implementations/vanilla/packages/ec-component-media-container/ec-component-media-container.scss

```diff
@@ -0,0 +1,33 @@
+/**
+ * Media container component
+ * @define media-container
+ */
+
+// Import base and generic
+@import '@ecl/ec-base/ec-base';
+
+@mixin ecl-media-container() {
+  .ecl-media-container {
+    background-color: $ecl-color-grey-10;
+    display: flex;
+    flex-direction: column;
+    margin: 0;
+  }
+
+  .ecl-media-container__media {
+    display: block;
+    height: auto;
+    object-fit: cover;
```

Comment:
`object-fit` is not supported by IE and has partial support on Edge (`<img>` only) https://caniuse.com/#search=object-fit

but the results look ok